openpype/hosts/max/api/plugin.py

```diff
@@ -11,6 +12,60 @@
 from openpype.lib import BoolDef
 from .lib import imprint, read, lsattr
 
+MS_CUSTOM_ATTRIB = """attributes "openPypeData"
+(
+    parameters main rollout:OPparams
+    (
+        all_handles type:#maxObjectTab tabSize:0 tabSizeVariable:on
+    )
+
+    rollout OPparams "OP Parameters"
+    (
+        listbox list_node "Node References" items:#()
+        button button_add "Add Selection"
+
+        fn node_to_name the_node =
+        (
+            handle = the_node.handle
+            obj_name = the_node.name
+            handle_name = obj_name + "<" + handle as string + ">"
+            return handle_name
+        )
+
+        on button_add pressed do
+        (
+            current_selection = selectByName title:"Select Objects To Add To
+            Container" buttontext:"Add"
+            temp_arr = #()
+            i_node_arr = #()
+            for c in current_selection do
+            (
+                handle_name = node_to_name c
+                node_ref = NodeTransformMonitor node:c
+                append temp_arr handle_name
+                append i_node_arr node_ref
+            )
+            all_handles = i_node_arr
+            list_node.items = temp_arr
+        )
+
+        on OPparams open do
+        (
+            if all_handles.count != 0 do
+            (
+                temp_arr = #()
+                for x in all_handles do
+                (
+                    print(x.node)
+                    handle_name = node_to_name x.node
+                    append temp_arr handle_name
+                )
+                list_node.items = temp_arr
+            )
+        )
+    )
+)"""
```

Comment:
should we store this script to somewhere more suitable? This probably needs to be used for the containerise function when refactoring the loader.

Reply:
it is actually used only at one place in `MaxCreatorBase`. There isn't any other use for it, loading is using metadata as before,  this is only for publishing instances